TrackletGraph.py

```diff
@@ -212,6 +227,9 @@ def populate_bitwidths(mem,hls_dir): # FIXME this information should be parsed f
         elif (    mem.mtype == "AllProj" or mem.mtype == "VMStubsME"
                or mem.mtype == "AllStubs" or mem.mtype == "TrackletParameters"):
             mem.bxbitwidth = 3
+        elif (    mem.mtype == "TrackWord"
+               or mem.mtype == "BarrelStubWord" or mem.mtype == "DiskStubWord"):
+            mem.bxbitwidth = 0
```

Comment:
Add comment at end of "mem.bxbitwidth = 0" explaining that this is "FIFO memory"

Reply:
Done.